src/SpendPermissionManager.sol

```diff
@@ -220,7 +237,7 @@ contract SpendPermissionManager is EIP712 {
     /// @notice Spend tokens using a spend permission, transferring them from `account` to `spender`.
     ///
     /// @param spendPermission Details of the spend permission.
-    /// @param value Amount of token attempting to spend (wei).
```

Comment:
i removed wei because technically this would work with ERC20s that use nonstandard decimals and as such the unit of account would be different.  spenders will have to assume they're operating in the smallest unit of account available for the token they're interacting with but that's not necessarily 1/10^18th worth